### DISCUSSION TITLE
Stuck on "Falling asleep" and being online always

### DISCUSSION THREAD

**Discussion by dhruvarora2** [teslamate-org/teslamate#1760] [Q&A]
Hi I have just installed the TeslaMate and it looks amazing but I am facing issues where its failing to put the car to sleep mode. 
PS: I am also using Tezlab Application and have its Deep sleep assist on. 

I am attaching logs: 

[error_logs.txt](https://github.com/adriankumpf/teslamate/files/6718950/error_logs.txt)
![Screen Shot 2021-06-25 at 4 46 20 PM](https://user-images.githubusercontent.com/16277682/123488106-e2377380-d5d4-11eb-97bd-b1c4eae39221.png)

**Comment by dhruvarora2** [comment]
Facing this post disabling  TezLab  too.

**Comment by adriankumpf** [comment]
Thanks for attaching the logs. I don't see any indicators of a TeslaMate bug in there. From 2021-06-25 11:21 to 2021-06-25 16:13 TeslaMate could not query your car because it was reported to be unavailable by the API.

In general, TeslaMate does not prevent your car from falling asleep and it will never send any commands to wake it up etc. There are dozens of existing [issues](https://github.com/adriankumpf/teslamate/issues?q=falling+asleep) and [discussions](https://github.com/adriankumpf/teslamate/discussions?discussions_q=falling+asleep+) on this topic. Please take a look there first. I'll move this issue to a discussion for now.

**Reply by dhruvarora2** [reply]
Thank you for looking into it. May I know what does teslamate does when it’s in Falling asleep state (prior to actually sleep state). Do you think we should add a retrial method after 5 mins which will cancel falling asleep and try again?
